### ISSUE TITLE
WaitCtx / test/integration test flake 

### ISSUE THREAD

**Issue by alexandreLamarre** [rancher/opni#1698]
https://drone-publish.rancher.io/rancher/opni/5169/1/3

```
  In [AfterAll] at: /src/pkg/util/waitctx/context.go:154 @ 09/07/23 18:51:41.233


  {}


  Full Stack Trace

    github.com/rancher/opni/pkg/util/waitctx.restrictive.WaitWWithTimeout({}, {0x2b2cfc8, 0xc0014d62d0}, 0xc001ddddf0?, {0xc0008b09f8, 0x1, 0x1})

    	/src/pkg/util/waitctx/context.go:154 +0x394

    github.com/rancher/opni/pkg/test.(*Environment).Stop(0xc0009dbc80)

    	/src/pkg/test/environment.go:473 +0x12f

    github.com/rancher/opni/test/integration_test.ExpectGracefulExamplePluginShutdown(0xc0009dbc80)

    	/src/test/integration/integration_suite_test.go:23 +0x16d

    github.com/rancher/opni/test/integration_test.glob..func10.2()

    	/src/test/integration/managementapi_user_access_test.go:44 +0x3c
```

**Comment by kralicky** [comment]
Fixed by [`20d9370` (#1692)](https://github.com/rancher/opni/pull/1692/commits/20d93705d48732c93f5b49308d64e55630f87aca)

**Comment by alexandreLamarre** [comment]
Closed by https://github.com/rancher/opni/pull/1749
